### PR TITLE
Homework20: Andrei Kulbitski

### PR DESCRIPTION
**Pull request by kulbitski-andrei**
In case if you get multiple PR review requests, sorry.
There was a lot of issues with raff, mypy and other checks.

### CODE REVIEW

File: modules_for_testing/bank_module.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# # TEST CASES
+# assert (client_socrates.money_amount == 40.0
+#         and client_socrates.currency == "BYN")
+# assert (client_plato.money_amount == 3.75
+#         and client_plato.currency == "EUR")
+# assert (client_aristotle.money_amount == 5
+#         and client_aristotle.currency == "USD")
+#
+# # TEST CASES
+# assert bank_pko.deposit(client_socrates, 5) == Decimal("84.29")
```
Comment (Stml89) [inline modules_for_testing/bank_module.py:113]: If you do not want to execute something when importing a module, put it under `if __name__ == __main__`

File: modules_for_testing/library_module.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        # print(f"В библиотеку записался новый читатель: {self.name}")
+
+    def reserve_book(self, book):
+        """Reserving a book to a particular reader
+        prevents reserving/taking this book by other reader"""
+
+        if book.status == "Reserved" and book.reserved_by != self.name:
+            # print(f"Невозможно зарезервировать книгу '{book.title}' "
+            #       f"для {self.name} - она уже зарезервирована "
+            #       f"другим читателем!")
```
Comment (Stml89) [inline modules_for_testing/library_module.py:44]: Why did you decide to comment this code? This log will be visible in the test, but use logging module.

File: modules_for_testing/bank_module.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+from decimal import Decimal
+
+formatter = logging.Formatter(
+    '\n%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+
+logger = logging.getLogger(__name__)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
```
Comment (Stml89) [inline modules_for_testing/bank_module.py:13]: You can use 1 logger across the application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: modules_for_testing/library_module.py (hunk 2 above)
Comment (kulbitski-andrei) [reply modules_for_testing/library_module.py]: Fixed

File: modules_for_testing/bank_module.py (hunk 3 above)
Comment (kulbitski-andrei) [reply modules_for_testing/bank_module.py]: Reworked project, so now it has separate module with logging setup and other modules import it

**Review by Stml89 — APPROVED** [review]

**Pull request merged**